### PR TITLE
Feature/components

### PR DESCRIPTION
**Pull request by mcrd25**
#Milestone 2
Created Following Components:
- App
- Display
- Button
- Button Panel

`App` renders `Display` and `Button Panel`

***Please note that I disabled eslint-disable-next-line react/prefer-stateless-function rule for eslint temporarily with an in-line comment because of the need of the project => required to use render function for App Component***

### CODE REVIEW

File: src/components/Display.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+const Display = (props) => {
+  const {
+    value,
+  } = props;
+  return (
+    <div className="calc-display">
+      {value}
+    </div>
+  );
+};
```
Comment (bolah2009) [inline src/components/Display.js]: [Optional]

Like `Button` component we could also destructure the `value` prop directly from the argument.

```diff
- const Display = (props) => {
+ const Display = ({ value }) => (
-  const {
-    value,
-  } = props;
-  return (
    <div className="calc-display">
      {value}
    </div>
  );
-};
```

File: src/index.js [hunk 2]
```diff
@@ -1,16 +1,17 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import PropTypes from 'prop-types';
+import App from './components/App';
+// import PropTypes from 'prop-types';
```
Comment (bolah2009) [inline src/index.js]: Consider deleting commented out codes. If you need to review a previous version of your code, use git history.

File: src/index.js [hunk 3]
```diff
@@ -1,16 +1,17 @@
...
-ReactDOM.render(<Title value="My React Calculator" />, document.getElementById('root'));
+// function Title(props) {
+//   const { value } = props;
+//   return (
+//     <h1>{value}</h1>
+//   );
+// }
+// Title.propTypes = {
+//   value: PropTypes.string.isRequired,
+// };
```
Comment (bolah2009) [inline src/index.js]: Let's delete this commented out code.

Comment (mcrd25) [reply src/index.js]: Yea, Thanks, oversight on my end! 😅

**Pull request merged**